components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/utils/LoggerUtils.java

```diff
@@ -169,4 +170,71 @@ private static Instant parseDateTime(String dateTimeString) {
         }
         return localDateTime;
     }
+
+    /**
+     * Util function to mask content.
+     *
+     * @param content Content that needs to be masked.
+     * @return masked content.
+     */
+    public static String maskContent(String content) {
+
+        if (StringUtils.isNotEmpty(content)) {
+            Pattern pattern = Pattern.compile(LogConstants.CONTENT_MASKING_REGEX);
+            content = pattern.matcher(content).replaceAll(LogConstants.MASKING_CHARACTER);
+        }
+        return content;
+    }
+
+    /**
+     * Util function to mask claim values except userid claim.
+     *
+     * @param claims Map of user claims.
+     * @return sanitized map of user claims.
+     */
+    public static Map<String, String> maskClaimValues(Map<String, String> claims) {
+
+        Map<String, String> sanitizedClaims = new HashMap<>();
+        if (claims != null && !claims.isEmpty()) {
```

Comment:
can't we use CollectionUtils.isEmpty() here?

Reply:
used MapUtils.isNotEmpty() instead as CollectionUtils.isEmpty can be used only on lists.